README.md

```diff
@@ -19,6 +19,8 @@ Edit Issue Submit   | While in Edit Issue Action: Press Submit → Wait for the
 Add Comment         | Go to Add Comment for an issue form → Wait for form to load → Enter comment text → Press Submit → Wait for the summary on view issue page to show
 Add Comment Submit  | While in Add Comment Action: Press Submit → Wait for the summary on view issue page to show
 Browse Projects     | Go to Browse Projects Page → Wait for the project list to load
+View Focused Comment| Go to Comment Tab Panel → Wait for focused comment to load
+View History Tab    | Go to History Tab Panel → Wait for all change history entries to load
```

Comment:
Nit: Maybe it would be good to mention that the actions are about sub-view of issue view. I'm not sure if I'd know this from the message without the context I currently have.